docs/conf.py

```diff
@@ -32,14 +32,85 @@
 # ones.
 extensions = [
     'sphinx.ext.ifconfig',
-    'breathe'
+    'breathe',
+    'exhale'
 ]
 
+# Setup the `breathe` extension
 breathe_projects = { "NanoGUI": "./doxyoutput/xml" }
 breathe_default_project = "NanoGUI"
 
+# Setup the `exhale` extension
+exhale_args = {
+    ############################################################################
+    # These arguments are required.                                            #
+    ############################################################################
+    "containmentFolder":     "./api",
+    "rootFileName":          "library_root.rst",
+    "rootFileTitle":         "Library API",
+    "doxygenStripFromPath":  "../include",
+    ############################################################################
+    # Suggested optional arguments.                                            #
+    ############################################################################
+    "createTreeView":        True,
+    "exhaleExecutesDoxygen": True,
+    "exhaleDoxygenStdin": textwrap.dedent('''
```

Comment:
For the most part, all the other Doxygen stuff from the original `conf.py` are in the Exhale defaults.  We only need `INPUT` and `PREDEFINED` for the namespaces.  I still have no idea why Doxygen can't handle those.